docs/griptape-framework/structures/config.md

```diff
@@ -34,6 +34,18 @@ agent = Agent(
 )
 ```
 
+#### Google
```

Comment:
I am curious if these examples that are the exact same code w/ just a different class name are useful in the docs? They seem like clutter to otherwise show a fairly straightforward concept of passing a new object type.

Is there something new that Google/Gemini offers that would be worth a new example?

Reply:
There are a handful of places in the docs that have redundant examples like this. Agreed that they may be overkill, though I think I'd rather be consistent for now and consider a larger restructuring in the future.